docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -166,7 +166,7 @@ metrics:
     symbols:
       # List of symbols ('columns') to retrieve.
       # Same format as for a single OID.
-      # Each row in the table will emit these metrics.
+      # The value from each row (index) in the table will be collected `<TABLE_OID>.1.<COLUMN_ID>.<INDEX>.*`
```

Comment:
It would be:

```suggestion
      # The value from each row (index) in the table will be collected `<TABLE_OID>.1.<COLUMN_ID>.*`
```

or 
```suggestion
      # The value from each row (index) in the table will be collected `<TABLE_OID>.1.<COLUMN_ID>.<INDEX>`
```